src/features/Instructors/InstructorsPage/_test_/index.test.jsx

```diff
@@ -1,8 +1,11 @@
 import React from 'react';
 import { waitFor } from '@testing-library/react';
-import InstructorsPage from 'features/Instructors/InstructorsPage';
+import { IntlProvider } from 'react-intl';
```

Comment:
This should not be necessary to import because renderWithProviders is already wrap all component with that